test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua

```diff
@@ -0,0 +1,185 @@
+--
+-- Description:
+-- SDL doesn’t transfer not allowed "SetGlobalProperties"_request with valid ""autoCompleteList" param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>.
+-- 3. SDL respond <success = true, resultCode = "DISALLOWED"> to mobile app
+-----------------------------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+require('user_modules/AppTypes')
+
+--[[ Local Functions ]]
+local registerAppInterfaceParams =
+  {
+    syncMsgVersion =
+    {
+      majorVersion = 3,
+      minorVersion = 0
+    },
+    appName = "Media Application",
+    isMediaApplication = true,
+    languageDesired = 'EN-US',
+    hmiDisplayLanguageDesired = 'EN-US',
+    appHMIType = {"NAVIGATION"},
+    appID = "MyTestApp",
+    deviceInfo =
+    {
+      os = "Android",
+      carrier = "Megafon",
+      firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+      osVersion = "4.4.2",
+      maxNumberRFCOMMPorts = 1
+    }
+  }
+
+local function UpdatePreloadedJson_DisallowSetGlobalPropertiesRPC()
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*all") 
+  file:close()
+  local json = require("modules/json")
+  local data = json.decode(json_data)
+
+  if data.policy_table.functional_groupings["DataConsent-2"] then
+    data.policy_table.functional_groupings["DataConsent-2"] = nil
+  end
+
+  data.policy_table.functional_groupings["Base-4"].rpcs["SetGlobalProperties"] = nil 
+
+  data = json.encode(data)
+  file = io.open(pathToFile, "w")
+  file:write(data)
+  file:close()
+end
+
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+function Test.Precondition_StopSDL()
+  StopSDL()
+end
+
+function Test.Precondition_DeleteLogsAndPolicyTable()
+commonSteps:DeleteLogsFileAndPolicyTable()
+end
+
+function Test.Precondition_Backup_preloadedPT()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+end
+
+function Test.DisallowSetGlobalProperties()
+  UpdatePreloadedJson_DisallowSetGlobalPropertiesRPC()
+end
+
+--[[Test]]
+commonFunctions:newTestCasesGroup("Test")
```

Comment:
@Aleksandriys, functions:
_StartSDL_With_DISALLOWED_SetGlobalProperties()
_initHMI()
_initHMI_onReady()
_ConnectMobile()
_CreateSession()
_RegisterApplication()
could be moved to preconditions also.